formatTest/typeCheckedTests/expected_output/features408re.4.08.0.re

```diff
@@ -0,0 +1,16 @@
+let (let.opt) = (x, f) =>
```

Comment:
wait, why can't we put this on the other file too?

Reply:
because it's not valid ocaml syntax :)